pulp_docker/app/registry.py

```diff
@@ -11,6 +12,7 @@
 from pulp_docker.app.models import DockerDistribution, Tag
 from pulp_docker.constants import MEDIA_TYPE
 
+SERVICE_NAME = 'v2'
```

Comment:
can we re-use variable defined in the auth_service? so we do no hardcode this twice?

Reply:
Sure, a name of the service may be provided in pulp_docker/app/settings.py.